ios/sidedishApp/sidedishApp/Presentation/UI/ViewControllers/SidedishViewController.swift

```diff
@@ -0,0 +1,146 @@
+        self.collectionView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        
+        let mainCellRegistration = UICollectionView.CellRegistration<SidedishCell, DataItem> { cell, indexPath, item in
+            if case .main(let item) = item {
+                cell.configureCell(item: item)
+            }
+        }
+        
+        let soupCellRegistration = UICollectionView.CellRegistration<SidedishCell, DataItem> { cell, indexPath, item in
+            if case .soup(let item) = item {
+                cell.configureCell(item: item)
+            }
+        }
+        
+        let sideCellRegistration = UICollectionView.CellRegistration<SidedishCell, DataItem> { cell, indexPath, item in
+            if case .side(let item) = item {
+                cell.configureCell(item: item)
+            }
+        }
+        
+        self.dataSource = UICollectionViewDiffableDataSource(collectionView: self.collectionView, cellProvider: { (collectionView, indexPath, model) -> UICollectionViewCell? in
+            guard let sectionKind = Section(rawValue: indexPath.section) else { return UICollectionViewCell() }
+            
+            switch sectionKind {
+            case .main:
+                return collectionView.dequeueConfiguredReusableCell(using: mainCellRegistration, for: indexPath, item: model)
+            case .soup:
+                return collectionView.dequeueConfiguredReusableCell(using: soupCellRegistration, for: indexPath, item: model)
+            case .side:
+                return collectionView.dequeueConfiguredReusableCell(using: sideCellRegistration, for: indexPath, item: model)
+            }
+        })
+        
+        let sectionHeaderRegistration = UICollectionView.SupplementaryRegistration<SectionHeader>(elementKind: SupplementaryElementKind.sectionHeader) { header, kind, indexPath in
+            guard let sectionKind = Section(rawValue: indexPath.section) else { return }
+        
+            header.label.text = sectionKind.sectionHeader
+        }
+        
+        self.dataSource.supplementaryViewProvider = { collectionView, kind, indexPath in
+            switch kind {
+            case SupplementaryElementKind.sectionHeader:
+                return collectionView.dequeueConfiguredReusableSupplementary(using: sectionHeaderRegistration, for: indexPath)
+            default:
+                return nil
+            }
+        }
+    }
+    
+    private func fetchMainData(path: String) {
+        self.fetchData(fetchData: self.sidedishViewModel.fetchMainData, path: path)
+    }
+    
+    private func fetchSoupData(path: String) {
+        self.fetchData(fetchData: self.sidedishViewModel.fetchSoupData, path: path)
+    }
+    
+    private func fetchSideData(path: String) {
+        self.fetchData(fetchData: self.sidedishViewModel.fetchSideData, path: path)
+    }
+    
+    private func fetchData(fetchData handler: FetchDataHandler, path: String) {
```

Comment:
```suggestion
    private func fetchData(handler: FetchDataHandler, path: String) {
```
=> 메소드 이름과 중복되네요, 해당 부분도 제거하면 더 좋을 것 같습니다